packages/javascript-sdk/src/token-manager/index.ts

```diff
@@ -105,33 +107,31 @@ abstract class TokenManager {
      * and return acquired tokens
      */
     if (options?.query?.code && options?.query?.state) {
-      const storedString = sessionStorage.getItem(storageKey);
-      sessionStorage.removeItem(storageKey);
-      const storedValues: { state: string; verifier: string } = JSON.parse(storedString as string);
+      const { state, verifier } = getStoredAuthUrlValues(clientId, prefix);
 
-      return await this.tokenExchange(options, storedValues);
+      if (state === undefined || verifier === undefined) {
+        throw new Error(
```

Comment:
smart. i wonder if we should add more like a "link" to a doc page (which we should add) that explains this exact problem which happens enough to get multiple tickets for.

Reply:
@forgerock-chris we get a ticket like this alot. usually relating to an email suspend node that when they click a the link in the email, they get errors like this. (but worse because justin added the context needed here)

I think we can add something somewhere to the docs that expands on how to resolve this (because its just using the existing session cookie to restart the flow).